template/user.html

```diff
@@ -0,0 +1,29 @@
+<link rel="import" href="../bower_components/app-storage/app-localstorage/app-localstorage-document.html">
+
+<dom-module id="papan-user-data">
+  <template>
+    <app-localstorage-document
+      key="papan-user-data"
+      data="{{user}}">
+    </app-localstorage-document>
+  </template>
+  <script>
+    Polymer({
+      is: 'papan-user-data',
+
+      properties: {
+        user: {
+          type: Object,
+          value: function() {
```

Comment:
Why using a function as value and not ¨directly¨give the value, e.g. value: { username: '', islogged: false, isadmin: false}? Do you always want a new object here?